RecoEgamma/EgammaPhotonProducers/python/photons_cfi.py

```diff
@@ -80,7 +80,13 @@
     RecHitSeverityToBeExcludedEE = cleanedHybridSuperClusters.RecHitSeverityToBeExcluded,
 )
 
-photonsFromMultiCl = photons.clone(
-  photonProducer = 'photonCoreFromMultiCl'
-)
-
+photonsFromMultiCl = photons.clone()
+# Change underlying C++ type, as ugly as it could be, and add required
+# configuration parameters
```

Comment:
what was wrong with cloning from ootPhotons, which would require only 1 line change in the inputs?